frontend/src/components/dashboard/SubdomainPicker.module.scss

```diff
@@ -27,7 +27,7 @@
 
   a {
     @include text-body-sm;
-    color: $color-purple-60;
+    color: $color-blue-50;
 
     &:hover,
     &:focus {
```

Comment:
If the regular colour is blue, should the hover state be dark blue instead of dark purple as well? (IIRC that's what we do elsewhere.)

Reply:
Ah yes, good callout!